connmgr/dynamicbanscore_test.go

```diff
@@ -66,3 +67,15 @@ func TestDynamicBanScoreReset(t *testing.T) {
 		t.Errorf("Failed to reset ban score.")
 	}
 }
+
+// TestDynamicBanScoreString
+func TestDynamicBanScoreString(t *testing.T) {
+	var bs DynamicBanScore
+	base := time.Now()
+
+	r := bs.increase(100, 50, base)
+	if r != 150 {
+		t.Errorf("Unexpected result %d after ban score increase.", r)
+	}
+	fmt.Println(bs.String())
```

Comment:
Rather than using fmt here, you could do `t.Log` or `t.Logf` and I think it should still work.

Reply:
👍OK. https://github.com/btcsuite/btcd/pull/1509/commits/c982d9b90643620b66f8758c4dcb48ef491c3547